improver/nowcasting/pysteps_advection.py

```diff
@@ -129,7 +130,7 @@ def _calculate_displacement(cube, interval, gridlength):
         displacement = np.array([udisp, vdisp])
         return displacement
 
-    def _reformat_analysis_cube(self):
+    def _reformat_analysis_cube(self, attribute_changes):
```

Comment:
The docstring for this method should probably be updated with an `Args` section for the `attribute_changes` argument. What do you think?

Reply:
I don't think it's necessary or particularly desirable to add tons of detail to private class method docstrings.  It's described in the `process` method (the public interface) what this argument does, and I didn't see any need to duplicate that here.